api.md

```diff
@@ -1832,31 +1832,68 @@ DROP MATERIALIZED VIEW contagg_view;
+Refresh all buckets of a continuous aggregate between two points in
 time. 
 
-The function expects the parameter values to have the same time type
-as used in the continuous aggregate's time bucket expression (e.g., if
-the time bucket specifies in `timestamptz`, then the start and end time
-supplied should also be `timestamptz`).
+The refresh will only materialize at the granularity of a bucket, as
+determined by the continuous aggregate view query's `time_bucket`
+expression. Therefore, only buckets that are completely covered by the
+_refresh window_ will be updated, and the window should account for
+any time zone shifts when applicable (note that a continuous aggregate
+using the `DATE` or `TIMESTAMP WITH TIME ZONE` type aligns with the
+UTC time zone).
+
+The function expects the window parameter values to have the same time
+type as used in the continuous aggregate's time bucket expression
+(e.g., if the time bucket specifies in `TIMESTAMP WITH TIME ZONE`,
+then the start and end time supplied should also be `TIMESTAMP WITH
+TIME ZONE`).
+
 
 #### Required Arguments [](refresh_continuous_aggregate-required-arguments)
 
 |Name|Description|
 |---|---|
 | `continuous_aggregate` | (REGCLASS) The continuous aggregate to refresh. |
-| `window_start` | Start of the window to refresh, has to be before `window_end`. `NULL` is eqivalent to `MIN(timestamp)` of the hypertable. |
-| `window_end` | End of the window to refresh, has to be after `window_start`. `NULL` is eqivalent to `MAX(timestamp)` of the hypertable. |
+| `window_start` | Start of the window to refresh (inclusive), has to be before `window_end`. `NULL` is equivalent to refreshing from the beginning of time. |
+| `window_end` | End of the window to refresh (exclusive), has to be after `window_start`. `NULL` is equivalent to refreshing to the end of time. |
+
+#### Errors
+
+An error is given if:
+
+- The refresh window does not cover a full bucket. Note that, even
+  though the window might be one bucket in size, it might not align
+  with the start and the end of a bucket in the continuous aggregate.
+- The function is called in a transaction block.
+  
 
 #### Sample Usage [](refresh_continuous_aggregate-examples)
 
-Refresh the continuous aggregate `conditions` between `2020-01-01` and
-`2020-02-01` exclusive.
+A continuous aggregate `conditions` has a bucket size of 1 day. To
+refresh `conditions` between `2020-01-01` and `2020-02-01` exclusive,
+do:
 
 ```sql
 CALL refresh_continuous_aggregate('conditions', '2020-01-01', '2020-02-01');
 ```
 
+To align the start and end of the refresh window with the bucket time
+zone, specify UTC. For instance, to refresh only 1 day/bucket:
+
+
+```sql
+CALL refresh_continuous_aggregate('conditions', '2020-01-01 00:00 UTC', '2020-01-02 00:00 UTC');
+```
+
+To ensure that at least one bucket is refreshed, without accounting
+for time zone, specify a window of at least two buckets:
```

Comment:
I'm honestly not sure how to do it, but it feels like this "at least two buckets" comment needs to be highlighted in some way. Assuming you're putting something together for the policy... maybe just link there??

Not a show stopper at all, just about to record examples of this and the wider your bucket is the more this will probably impact you.

Reply:
Oops... just keep scrolling Ryan... my apologies.